src/controllers/user/franceconnect.ts

```diff
@@ -212,16 +214,16 @@ export function getFranceConnectLogoutCallbackControllerFactory(
               operator: "===",
             }),
           },
-        );
-      }
+        ),
```

Comment:
i do not get the necessity of manually calling `next` on error as we already call it in the catch below